### ISSUE TITLE
Console logs are coming after upgrading to version 1.5.1

### ISSUE THREAD

**Issue by kakadiadarpan** [localForage/localForage#750]
I'm getting following logs after upgrading to version 1.5.1:

<img width="870" alt="screen shot 2017-10-10 at 21 45 38" src="https://user-images.githubusercontent.com/10805593/31397763-7cd2ec1e-ae04-11e7-8378-c45a56cc2a18.png">

My configuration for localforage is as below:
```
import localforage from "localforage";
localforage.config({
  version: 1.0,
  description: "Personal DB"
});
const instance1 = localforage.createInstance({
  name: "db-instance1",
  storeName: "instance1"
});
const instance2 = localforage.createInstance({
  name: "db-instance2",
  storeName: "instance2"
});
const instance3 = localforage.createInstance({
  name: "db-instance3",
  storeName: "instance3"
});
```

**Comment by tofumatt** [comment]
Ah, interesting, are you defining any custom drivers? If not it seems we have a bug and shouldn't be logging here.

**Comment by kakadiadarpan** [comment]
@tofumatt No, I'm not using any custom driver. In my previous comment, I logged the config that I'm using.

**Comment by thgreasi** [comment]
@kakadiadarpan can you test #751. If you can confirm that it work I will try to release is today or early tomorrow.

**Comment by kakadiadarpan** [comment]
@thgreasi Sure, I'll do that.

**Comment by kakadiadarpan** [comment]
@thgreasi Yes, #751 fixes this issue. Thanks!

**Comment by thgreasi** [comment]
Released a fix as 1.5.2 , thanks for the detailed report!
Happy instance creation!

**Comment by kakadiadarpan** [comment]
Wow! That was fast. :clap:
